doc/generators.md

```diff
@@ -1,11 +1,10 @@
 <!-- AUTOGENERATED. See 'doc/build.jl' for source. -->
 ## Random Graphs
 *LightGraphs.jl* implements three common random graph generators:
-### erdos_renyi
 ```
-erdos_renyi(n::Integer, ne::Integer)
-erdos_renyi(n::Integer, p::Real)
+erdos_renyi(n::Integer, p::Real; is_directed=false, seed::Integer=-1)
 ```
```

Comment:
...I think this is a problem. We have two very different methods for `erdos_renyi()` - one takes two Integers, and the other takes an int and a real. The PR will remove very useful information about one of the more common methods of creating an E-R graph.